Formula/openjdk.rb

```diff
@@ -33,87 +34,90 @@ class Openjdk < Formula
   on_linux do
     depends_on "pkg-config" => :build
     depends_on "alsa-lib"
+    depends_on "cups"
+    depends_on "fontconfig"
+    depends_on "libx11"
+    depends_on "libxext"
+    depends_on "libxrandr"
+    depends_on "libxrender"
+    depends_on "libxt"
+    depends_on "libxtst"
+    depends_on "unzip"
+    depends_on "zip"
   end
 
   # From https://jdk.java.net/archive/
   resource "boot-jdk" do
     on_macos do
       if Hardware::CPU.arm?
-        url "https://download.java.net/java/early_access/jdk16/31/GPL/openjdk-16-ea+31_osx-x64_bin.tar.gz"
-        sha256 "8e4a8fdd2d965067bdb56e1a72c7c72343d571b371ac61eee74d9e71bbef63e8"
+        url "https://download.java.net/java/early_access/jdk17/22/GPL/openjdk-17-ea+22_macos-aarch64_bin.tar.gz"
+        sha256 "4532256e0ff66e31260c2037c92a183556d98c37f6e183319df0a8bccda1a95f"
```

Comment:
We ended up hitting the JDK failure for macOS 11.4 
Maybe try updating the boot JDK also
https://download.java.net/java/early_access/jdk17/24/GPL/openjdk-17-ea+24_macos-aarch64_bin.tar.gz